airsonic-main/src/main/java/org/airsonic/player/service/sonos/SonosHelper.java

```diff
@@ -116,7 +116,7 @@
 
```

Comment:
There are only irrelevant stylistic changes here, please put those into an other pull-request.

Reply:
That would be my IDE auto-boyscouting.